### PR TITLE
fix: App does not crash on bookmarking sessions in TrackSessionActivity

### LINKED ISSUE #2277: App crashes while bookmarking sessions in TrackSessionsActivity
**Actual Behaviour**

App crashes while bookmarking sessions in TrackSessionsActivity

**Expected Behaviour**

App should not crash

**Steps to reproduce it**

Open tracks fragment.
Select Open Tech.
BookMark sessions

**LogCat for the issue**

01-30 22:50:01.429 547-547/org.fossasia.openevent E/DateConverter: java.lang.NullPointerException: text must not be null

                                                                    java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String org.fossasia.openevent.data.Track.getName()' on a null object reference

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by nikit19**
Fixes #2277 

Changes: Added null checks to prevent crashes

Screenshots for the change: N/A

### CODE REVIEW

File: android/app/src/main/java/org/fossasia/openevent/activities/TrackSessionsActivity.java [hunk 1]
```diff
@@ -196,12 +196,15 @@ public void setUpcomingSession() {
...
-                track = session.getTrack().getName();
-                color = session.getTrack().getColor();
-                break;
+            if (session.getStartDate() != null) {
+                ZonedDateTime start = DateConverter.getDate(session.getStartsAt());
+                if (start.isAfter(current)) {
+                    upcomingTitle = session.getTitle();
+                    if (session.getTrack().getName() != null)
+                        track = session.getTrack().getName();
+                    color = session.getTrack().getColor();
```
Comment (dr0pdb) [inline android/app/src/main/java/org/fossasia/openevent/activities/TrackSessionsActivity.java]: What if the color or the title is also null ?

Comment (nikit19) [reply android/app/src/main/java/org/fossasia/openevent/activities/TrackSessionsActivity.java]: True, added null checks for them too

**Review by iamareebjamal — DISMISSED** [review]

**Comment by nikit19** [comment]
@iamareebjamal it says This branch has conflicts that must be resolved but when I pull the latest changes in my local repository I don't get any merge conflicts.

**Comment by iamareebjamal** [comment]
Fetch the latest development branch and rebase against it

**Comment by nikit19** [comment]
@iamareebjamal Done

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**